app/presenters/tree_builder.rb

```diff
@@ -349,7 +349,7 @@ def x_get_tree_objects(parent, options, count_only, parents)
                         when DialogGroup         then x_get_tree_dialog_group_kids(parent, count_only, options[:type])
                         when DialogTab           then x_get_tree_dialog_tab_kids(parent, count_only, options[:type])
                         when ExtManagementSystem then x_get_tree_ems_kids(parent, count_only)
-                        when EmsFolder           then if parent.is_datacenter
+                        when EmsFolder           then if parent.kind_of?(Datacenter)
```

Comment:
Might be cleaner to break this up into 2 separate `when` conditions...put Datacenter first and it will work out.